components/Layout/index.ts

```diff
@@ -1,6 +1,12 @@
 import { Header } from "./Header";
 import { Footer } from "./Footer";
+import { Form } from "./MsgForm";
+import { PopUp } from "./popUp";
+import {UserInput} from "./MsgForm";
 
 
 export { Header}
-export { Footer } 
\ No newline at end of file
+export { Footer } 
```

Comment:
its ok, but you can write all as list in export 

export { footer, header, ....}